reference/UQ Handbook Chapter.ipynb

```diff
@@ -20,7 +20,7 @@
    "cell_type": "markdown",
```

Comment:
Yes, please go ahead and add some MCMC background. Maybe say "skip to section X if you're already familiar with MCMC" or something to that effect.